cmd/argo/commands/lint.go

```diff
@@ -37,32 +38,41 @@ func NewLintCommand() *cobra.Command {
 						return err
 					}
 				}
-				fmt.Printf("%s is valid\n", file)
+				fmt.Printf("%s is valid \u2713\n ", file)
```

Comment:
We don't want unicode characters in our messages.
```suggestion
				fmt.Printf("%s is valid\n ", file)
```